servo-build-dependencies/init.sls

```diff
@@ -35,16 +36,43 @@ servo-dependencies:
       - xpra
       - xserver-xorg-input-void
       - xserver-xorg-video-dummy
+      {% elif grains['os'] in ('CentOS', 'Fedora') %}
+      - bzip2-devel
+      - cabextract
+      - curl
+      - dbus-devel
+      - expat-devel
+      - fontconfig-devel
+      - freeglut-devel
+      - freetype-devel
+      - gcc-c++
+      - glib2-devel
+      - gperf
```

Comment:
It's in the list of Servo prerequisites at https://github.com/servo/servo/#on-debian-based-linuxes; I forget which dependency in particular requires it.